torch/csrc/api/include/torch/data/datasets/base.h

```diff
@@ -22,6 +22,13 @@ MapDataset<D, T> map(D, T); // NOLINT
 } // namespace data
 } // namespace torch
 
+namespace torch {
+namespace serialize {
```

Comment:
Not sure whey we need this change?

Reply:
I think it is git that is confused about the base line. It should disappear once we submit the official pr to fb.